src/platform/web.rs

```diff
@@ -65,8 +65,76 @@ pub trait WindowExtWebSys {
     /// For example, by default using the mouse wheel would cause the page to scroll, enabling this
     /// would prevent that.
     ///
+    /// Specifically, this will call `event.preventDefault()` for the following event types:
+    /// `touchstart`, `wheel`, `contextmenu`, `pointerdown`, `pointermove`, `keyup`, and `keydown`.
+    ///
+    /// For fine-grained control over which events call `event.preventDefault()`, call
+    /// `Window::set_prevent_default(false)` and add an event listener on the canvas for each of
+    /// the above event types that selectively calls `event.preventDefault()`.
+    ///
     /// Some events are impossible to prevent. E.g. Firefox allows to access the native browser
     /// context menu with Shift+Rightclick.
+    ///
+    /// # Example
+    /// This example calls `event.preventDefault()` for all relevant events except for ctrl-c,
+    /// ctrl-x, and ctrl-v.
+    /// ```
+    /// # fn add_prevent_default_listeners(canvas: HtmlCanvasElement) {
```

Comment:
It would be good to actually make a full example with `EventLoop::new()` and such.
Will be adding cross-compilation testing for rustdoc into the CI soon.

Doesn't need much, just create a `Window`, use [`WindowBuilder::with_append()`](https://docs.rs/winit/0.29.10/wasm32-unknown-unknown/winit/window/struct.WindowBuilder.html#method.with_append) and get the canvas.

Reply:
Done!